Sming/Arch/Host/Tools/ci/build.setup.sh

```diff
@@ -11,11 +11,13 @@ if [ "$DIFFS" != "" ]; then
 fi
 
 # Make deployment keys, etc. available
-set +x
-openssl aes-256-cbc -d -a -iter 100 -in $CI_BUILD_DIR/.ci/secrets.sh.enc -out /tmp/secrets.sh -pass pass:$SMING_SECRET
-source /tmp/secrets.sh
+if [ -z "$CI_PULL_REQUEST" ]; then
```

Comment:
See this link: https://help.appveyor.com/discussions/questions/2615-secure-variables-for-pull-request

Reply:
Yes, the check is necessary though to prevent the decryption failing which kills the build.